lib/routes/admin-api/state.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const Controller = require('../controller');
+const { ADMIN } = require('../../permissions');
+const extractUser = require('../../extract-user');
+const { handleErrors } = require('./util');
+const YAML = require('js-yaml');
+const moment = require('moment');
+const multer = require('multer');
+const upload = multer({ limits: { fileSize: 5242880 } });
```

Comment:
so the limit i 5MB? Why did you choose this number?

Reply:
the file is kept in memory, so I chose a number that wouldn't impact memory usage greatly. 
Maybe this should be exposed as a config?

Reply:
Yes, sounds reasonable. I would be very surprised if anyone need more than 5MB, but you never know.